l10n_br_fiscal/views/document_fiscal_line_mixin_view.xml

```diff
@@ -404,7 +411,10 @@
                                 attrs="{'invisible': [('tax_icms_or_issqn', '=', 'issqn')]}"
                             >
                   <group string="IPI">
-                      <field name="ipi_tax_id" />
+                      <field
+                                        name="ipi_tax_id"
+                                        attrs="{'readonly': [('rel_tax_calc', '=', 'tax_calc_manual')]}"
+                                    />
```

Comment:
@ygcarvalh o @mbcosta tem razão nesse comentario https://github.com/OCA/l10n-brazil/pull/1412#discussion_r672582721 !!
Esse PR tem falha conceitual grave hoje.

Vou pegar o examplo do IPI apenas mas o problema é generalizado a todos os impostos.
Hoje se vc seleciona um ipi_tax_id, o Odoo vai selecionar automaticamente um ipi_cst_id com um onchange aqui:
https://github.com/OCA/l10n-brazil/blob/12.0/l10n_br_fiscal/models/document_fiscal_line_mixin_methods.py#L574

So que se vc não puder editar mais o ipi_tax_id ('rel_tax_calc', '=', 'tax_calc_manual'), no seu PR vc  nao tem como escolher um ipi_cst_id correcto e bem como falou o @mbcosta , mesmo que o nao vai ele mesmo no mapping da NFe, o ipi_cst_id participa sim do mapping da NFe (aqui por examplo https://github.com/OCA/l10n-brazil/blob/12.0/l10n_br_nfe/models/document_line.py#L491 )

Logo o modo tax_calc_manual do seu PR é inusavel...

@mileo @ygcarvalh uma falha conceitual dessa apenas demostra que vcs nao usam esse codigo em produção...

Eu acho muita sacanagem @mileo a instancia como vc vinha para fazer o merge de uma versão ainda inferior a isso nessa PR https://github.com/OCA/l10n-brazil/pull/1311 empinando o narriz como se vc tivesse salvando a contabilidade da localização (e vai que muita outra coisa desse PR tb tava furada como irei te mostrar depois ainda) e depois eu e o @mbcosta a gente gasta 2h para revisar esse PR de 400 linhas para descobrir que vcs estão zoando da nossa cara com falha conceitual básica que torna a funcionalidade impossivel de usar hoje.

@mileo vc não é mais nenhum iniciante, o modulo l10n_br_fiscal tb não é mais um POC, a gente te aceitou no PSC do projeto por favor tentar honrar o voto de confiança senão pela proteção do projeto a gente vai ter que pedir de reverter isso infelizmente... Nosso tempo de revisão tb é precioso. Ontem vc mandou outro PR de mais de 500 linhas, como vc quer que a gente te leva a serio e gasta tempo com  suas propostas? Se fosse o único erro no qual vc insiste OK, mas o problema é que não é, bem longe disso, se a gente botar no lapis (espero que vc não me obriga a faze-lo) por vc a gente já teria matado o projeto 3x.

@ygcarvalh se vc for mexer mais nessa PR por favor tome cuidado em manter algo simples, porque a gente tem que se preocupar com o custo/beneficio de uma funcionalidade dessa. O modulo l10n_br_fiscal ja ta de sobre-peso (e em parte por conta de inclusão nao revisada da KMEE como de closing ou dfe que são coisas que com um minimo de competencia era perfeitamente possivel botar em modulos extra), então ve que antes de integrar mais 400 linhas a gente vai pensar 2 vezes. Se a gente ainda ve que o PR nem ta usável ai realmente á escolha é rapida...

cc @renatonlima @marcelsavegnago